### ISSUE TITLE
Error - Stalker Creepers 1.21.1NF - NeoForge

### ISSUE THREAD

**Issue by Ankeltor** [AtomicStryker/atomicstrykers-minecraft-mods#515]
The NeoForge version of the mod "stalkercreepers-1.21.1NF.jar" is not correct, it identifies itself as a Forge mod.

Error:
Line 27
[00:34:02.991] [main/WARN] [loading.moddiscovery.ModDiscoverer/SCAN]: Skipping jar. File /home/container/mods/QL_SS-ASG-Stalker_Creepers-NeoForge-MC1.21-v1.21.1NF.jar is for an old version of Minecraft Forge and cannot be loaded

I tested it without any other mod installed. I hope it helps.
[2024-08-03-1.log](https://github.com/user-attachments/files/16483106/2024-08-03-1.log)

**Comment by AtomicStryker** [comment]
No, i think thats just a log message they didnt port to NeoForge. I have updated Stalker Creepers NeoForge dependency for you:
https://github.com/AtomicStryker/atomicstrykers-minecraft-mods/commit/70e36919d5c1bd79e62c5b316ae96d51e28559e4

And pushed that to curseforge too. Should work? Thanks for reporting.

**Comment by Ankeltor** [comment]
The error persisted with this latest version.

Checking the files I noticed that there was no neoforge.mods.toml file in the META-INF folder.
I renamed the mods.toml to neoforge.mods.toml and changed some fields to match your Multimine mod. I tested it on client (NF 21.0.151) and server (NF 21.0.148), it loads correctly and seems to work without problems, both in single and multiplayer. 

The modified neoforge.mods.toml file: [neoforge.mods.zip](https://github.com/user-attachments/files/16487046/neoforge.mods.zip)
